components/brave_new_tab_ui/containers/newTab/settings/topSites.tsx

```diff
@@ -40,17 +64,44 @@ class TopSitesSettings extends React.PureComponent<Props, {}> {
             size='large'
           />
         </SettingsRow>
-        {
-          showCustomizedLink ?
-          (<SettingsRow>
-            <SettingsText>{getLocale('topSiteCustomLinksEnabled')}</SettingsText>
-            <Toggle
-              onChange={toggleCustomLinksEnabled}
-              checked={customLinksEnabled}
-              size='large'
-            />
-          </SettingsRow>) : null
-        }
+        <StyledTopSitesCustomizationSettings>
+          <StyledTopSitesCustomizationSettingsOption>
+            <StyledTopSitesCustomizationImageBorder
+              selected={favoritesSelected}
+            >
+              <StyledTopSitesCustomizationImage
```

Comment:
How about letting the CSS decide what to change in dark / light mode and just passing in a prop to `StyledTopSitesCustomizationImage` whether it is favorites or frecency selected? I'm suggesting this so that we can put the graphic decisions (and non-functional light/dark decisions) in the style file.

Reply:
Moved to css.